NEWS.md

```diff
@@ -1,3 +1,14 @@
+# New in snapd 2.61.1:
+* Added prerequisite tracker for self-contained snap sets
+* Fix auth.json access for login as non-root group ID
+* Fix incorrect remodelling conflict when changing track to older snapd version
+* Improved check-rerefresh message
+* Fix UC16/18 kernel/gadget update failure due volume mismatch with installed disk
+* Stop auto-import of assertions during install modes
+* Desktop interface exposes GetIdletime
+* Polkit interface support for new polkit versions
+* Fix to apply snapd changes in model assertion
```

Comment:
This line hardly is very descriptive, is it possible this could make more sense, or perhaps just remove it?

Reply:
Agree, will improve